src/Factory/RepositoryContainer.php

```diff
@@ -21,11 +21,12 @@ final class RepositoryContainer implements ContainerInterface
     private bool $rolesBuilt = false;
     private array $roles = [];
 
-    private array $instances;
+    private array $instances = [];
 
     public function __construct(ContainerInterface $rootContainer)
     {
         $this->rootContainer = $rootContainer;
+        $this->orm = $rootContainer->get(ORMInterface::class);
```

Comment:
ORM created in method `makeRepositoryList`. We can make property `$orm` is nulled.

Reply:
removed leftovers https://github.com/yiisoft/yii-cycle/pull/135/commits/5882f7d2b9a47c4fecd705e1fecb9314e96e1c41